
Thread #1 is the program's root thread

Thread #2 was created
   at 0x........: clone (in /...libc...)
   by 0x........: ...
   by 0x........: pthread_create@GLIBC_ (in /lib/libpthread...)
   by 0x........: pthread_create@* (hg_intercepts.c:...)
   by 0x........: main (tc06_two_races.c:26)

Possible data race during read of size 4 at 0x........ by thread #1
   at 0x........: main (tc06_two_races.c:31)
 This conflicts with a previous access by thread #2
   at 0x........: child_fn (tc06_two_races.c:14)
   by 0x........: mythread_wrapper (hg_intercepts.c:...)
   by 0x........: ...
   by 0x........: ...
 Location 0x........ is 0 bytes inside global var "unprot1"
 declared at tc06_two_races.c:9

Possible data race during read of size 4 at 0x........ by thread #1
   at 0x........: main (tc06_two_races.c:35)
 This conflicts with a previous access by thread #2
   at 0x........: child_fn (tc06_two_races.c:18)
   by 0x........: mythread_wrapper (hg_intercepts.c:...)
   by 0x........: ...
   by 0x........: ...
 Location 0x........ is 0 bytes inside global var "unprot2"
 declared at tc06_two_races.c:9

ERROR SUMMARY: 2 errors from 2 contexts (suppressed: 0 from 0)
